redfish_guide/topics/using-redfish-physical-infra.adoc

```diff
@@ -135,6 +135,92 @@ PXE server with the network boot parameters specified in the request for the
 servers defined in the request. It then restarts these servers into one-time
 network boot, initiating the provisioning.
 
+==== Configuring PXE
+
+In {product-title}, a PXE server represents a collection of services and files
+that, all together, enable computers to boot into Operating Systems from the
+network:
+
+* The {product-title} needs to be able to connect to the PXE server for both
+  reading and writing using either NFS or SAMBA protocol. This is also called
+  the depot type.
+* The physical servers' NICs that are connected to the network boot network
+  require a TFTP access for reading the initial executable file containing the
+  PXE itself, and the HTTP access for downloading the Operating Systems' files.
+
+The process of configuring the PXE Servers is similar to that explained in the
+*PXE Provisioning* section of the *Provisioning Virtual Machines and Instances*

```

Comment:
I'd go with 

```
<<PXE Provisioning>> section of the <<Provisioning Virtual Machines and Instances>>
```

so that PDF will crate links. In our local document the links won't work because we don't have those sections, but when they put it all together it _should_ just work

Reply:
Or not, as @tadeboro suggested there is no "one big PDF" available but rather number of small PDFs and you're refering to https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0-beta/pdf/provisioning_virtual_machines_and_instances/Red_Hat_CloudForms-5.0-Beta-Provisioning_Virtual_Machines_and_Instances-en-US.pdf so the reference can not work really.